### PR TITLE
Added a special variable controlling the make-hash-table :test specifier

### PR DESCRIPTION
**Pull request by Mr-Dispatch**

### CODE REVIEW

File: src/config.lisp [hunk 1]
```diff
@@ -64,4 +64,6 @@ is an plist and should be written as object.")
 
 (defvar *symbol-name-function* #'symbol-name)
 
+(defvar *read-default-ht-test* 'equal
```
Comment (yitzchak) [inline src/config.lisp:67]: Let's just call this `*read-hash-table-test*`. Please move it to to bottom of the other read dynamic variables.

File: src/read.lisp [hunk 2]
```diff
@@ -402,7 +402,8 @@ and formats used. The following arguments also control the behavior of the read.
                         ((:object-format *read-default-object-format*) *read-default-object-format*)
                         ((:float-format *read-default-float-format*) *read-default-float-format*)
                         ((:length *read-length*) *read-length*)
-                        ((:level *read-level*) *read-level*))
+                        ((:level *read-level*) *read-level*)
+                        ((:ht-test *read-default-ht-test*) *read-default-ht-test*))
```
Comment (yitzchak) [inline src/read.lisp:406]: Let's call the key `:hash-table-test`. Also, please update the documentation string.

**Review by yitzchak — CHANGES_REQUESTED** [review]
Just a few tweaks.

File: src/read.lisp (hunk 2 above)
Comment (Mr-Dispatch) [reply src/read.lisp]: Done. Also updated docstring for read-json.

File: src/config.lisp (hunk 1 above)
Comment (Mr-Dispatch) [reply src/config.lisp]: Done.

**Review by yitzchak — APPROVED** [review]

**Comment by yitzchak** [comment]
Thanks!

**Comment by Mr-Dispatch** [comment]
I realized I let the readme go stale, and tried to add the following:

```
- * hash-table-test - Value for the make-hash-table :test designator, in case
    *read-default-object-format* is set to :hash-table
```
to the end of read-related dynamic vars.

For some strange reason, git sees conflicts when I try to push to my own unchanged fork.
Could I trouble you to do it "manually" ?

Thanks.

**Comment by yitzchak** [comment]
Don't worry about the README. There are other things I need to add eventually.

**Pull request merged**